### PR TITLE
Propose RFC 0428: Prerequisites to Issue Rich Credential

### PR DESCRIPTION
**Pull request by brentzundel**
Rich schema prerequisites for issuance of credentials.
Signed-off-by: Brent <brent.zundel@gmail.com>

### CODE REVIEW

**Comment by brentzundel** [comment]
@ashcherbakov please let us know what you think of this.

File: features/0428-prepare-issue-rich-credential/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## Motivation
+
+To inform issuers of the steps they should take in order to make sure they have
+the necessary rich schema objects in place before they use them to issue
+credentials.
+
+## Tutorial
+
+###Rich Schema Credential Workflow
```
Comment (ashcherbakov) [inline features/0428-prepare-issue-rich-credential/README.md]:
```suggestion
### Rich Schema Credential Workflow
```

File: features/0428-prepare-issue-rich-credential/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+1. The issuer checks the ledger to see if the credential definition he wants to
+use is already present.
+   1. If not, the issuer checks the ledger to see if the mapping he wants to use
+   is already present.
+      1. If not, the issuer checks the ledger to see if the schemas he wants to
+      use are already present.
+         1. If not, anchor the context used by each schema to the ledger.
```
Comment (ashcherbakov) [inline features/0428-prepare-issue-rich-credential/README.md]: Should we clarify what we mean by anchoring here? Is it writing the full schema/mapping's content to the ledger, or just an anchor/digital_fingerprint/hash of the full content?

Comment (brentzundel) [reply features/0428-prepare-issue-rich-credential/README.md]: I think whether the full contents of the schema/mapping should be on the ledger versus a digital hash is determined by the user's trust model, so both should be allowed. I think using the more generic term `anchor` allows for either interpretation.

Comment (ashcherbakov) [reply features/0428-prepare-issue-rich-credential/README.md]: I agree as we are talking about an Aries RFC here. 
Once we write a ledger-specific RFC (Indy HIPE for example), then we may need to clarify the anchoring term in terms of supported transactions there.

**Pull request merged**